bobtemplates/plone_addon/src/+package.namespace+/+package.name+/interfaces.py.bob

```diff
@@ -2,13 +2,16 @@
 """Module where all interfaces, events and exceptions live."""
 
 from {{{ package.dottedname }}} import _
```

Comment:
@pbauer this leads to a code analysis violation since the _ import is unused. Not sure why travis did not complained about it.